.travis.yml

```diff
@@ -44,4 +44,12 @@ install:
   - docker-compose pull
 
 script:
-  - docker-compose run --use-aliases -e BOULDER_CONFIG_DIR="${BOULDER_CONFIG_DIR}" -e RUN="${RUN}" -e TRAVIS="${TRAVIS}" -e TRAVIS_PULL_REQUEST="${TRAVIS_PULL_REQUEST}" -e COVERALLS_TOKEN="${COVERALLS_TOKEN}" ${CONTAINER:-boulder} ./test.sh
+  - >-
```

Comment:
What does the `>-` syntax do?

Reply:
It's a folded scalar, it collapses all lines into a single one: https://yaml-multiline.info/

Reply:
Neat!